### PR TITLE
RHEL6 to only use its local shorthand content

### PR DESCRIPTION
**Pull request by mpreisler**
This is required for the yaml work.

### CODE REVIEW

**Comment by mpreisler** [comment]
Something is wrong with this, I will take a look at it tomorrow. If anybody has any ideas I am all ears.

**Comment by mpreisler** [comment]
There is a bunch of typos, I will get it fixed:
```
warning: failed to load external entity "/home/mpreisle/d/scap-security-guide/rhel6/xccdf/system/software/permissions.xml"
warning: failed to load external entity "/home/mpreisle/d/scap-security-guide/rhel6/xccdf/system/accounts/restrictions.xml"
```

File: rhel6/xccdf/services/sssd.xml [hunk 1]
```diff
@@ -0,0 +1,272 @@
+<Group id="sssd" prodtype="rhel6,rhel7">
```
[the remark this answers is not in the document]
Comment (mpreisler) [reply rhel6/xccdf/services/sssd.xml]: I think this should work because when it was including the shared XMLs it worked.

**Comment by mpreisler** [comment]
Seems to be all fixed, this is now ready for review.

**Comment by redhatrises** [comment]
@openscap-ci test this please

**Comment by redhatrises** [comment]
Ack.

**Pull request merged**